src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -613,7 +611,7 @@ void GcsActorManager::HandleGetAllActorInfo(rpc::GetAllActorInfoRequest request,
             absl::flat_hash_map<ActorID, rpc::ActorTableData>>(arena, std::move(result));
         size_t count = 0;
         size_t num_filtered = 0;
-        for (const auto &pair : *ptr) {
+        for (auto &pair : *ptr) {
```

Comment:
why changing this?

Reply:
so below at line 625 there's a const_cast for the UnsafeArenaAddAllocated call, shouldn't have const casts, and just taking the const away here allows use to remove the const cast